app/controllers/application_controller/ci_processing.rb

```diff
@@ -215,7 +215,8 @@ def ownership_update
   # Retire 1 or more VMs
```

Comment:
@lgalis, can you, please, remove the missleading comment and rename the variable `vms` to something that suggest that we deal here with different things?